### PR TITLE
chore: add doc and license file

### PR DESCRIPTION
**Pull request by gaetanmaisse**

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
+= Contributing to Gravitee.io
+
+You think Gravitee.io is awesome and want to contribute?
+Here are few guidelines that should help you get started.
+
+== Using GitHub issues
+
+We are using GitHub issues to track bugs and enhancements. Found a bug in the source code? Want to propose new features or enhancements?
+You can help us by submitting an issue in our https://github.com/gravitee-io/issues/issues[repository].
+Before submitting your issue, search the https://github.com/gravitee-io/issues/issues[issues archive] or the https://waffle.io/gravitee-io/release[backlog]; maybe your question has already been answered.
```
Comment (phiz71) [inline CONTRIBUTING.adoc]: I think waffle.io does not exist anymore

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+= Gravitee.io API Management
+
+image:https://img.shields.io/badge/License-Apache%202.0-blue.svg["License", link="https://github.com/gravitee-io/gravitee-api-management/blob/master/LICENSE"]
+image:https://circleci.com/gh/gravitee-io/gravitee-api-management.svg?style=svg["CircleCI", link="https://circleci.com/gh/gravitee-io/gravitee-api-management"]
+
+⚠️ Notes: This repository is under construction and doesn't contain all Gravitee.io API Management source code.
+It will be completed step by step by merging other API Management related repositories like `gravitee-gateway` or `gravitee-management-rest-api`.
+
+
+image:./assets/gravitee-logo-cyan.svg["Gravitee.io",400]
```
Comment (gaetanmaisse) [inline README.adoc:11]: Image should work when the PR will be merged

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
+= Gravitee.io API Management
+
+image:https://img.shields.io/badge/License-Apache%202.0-blue.svg["License", link="https://github.com/gravitee-io/gravitee-api-management/blob/master/LICENSE"]
+image:https://circleci.com/gh/gravitee-io/gravitee-api-management.svg?style=svg["CircleCI", link="https://circleci.com/gh/gravitee-io/gravitee-api-management"]
```
Comment (gaetanmaisse) [inline README.adoc:5]: Image should work when the PR will be merged and this repo open source

File: CONTRIBUTING.adoc (hunk 1 above)
Comment (gaetanmaisse) [reply CONTRIBUTING.adoc]: Indeed I will remove it

**Pull request merged**